axelrod/moran.py

```diff
@@ -389,14 +395,15 @@ class ApproximateMoranProcess(MoranProcess):
     Instead of playing the matches, the result is sampled
     from a dictionary of player tuples to distribution of match outcomes
     """
-    def __init__(self, players, cached_outcomes, mutation_rate=0.):
+    def __init__(self, players: List[Player], cached_outcomes: dict,
```

Comment:
Are you asking about the `turns` `prob_end` etc? In which case: no we don't. They are not used as all match results sampled from the `cached_outcomes`.

If you're asking about the `graphs`. I didn't want to tackle that here. It would be an easy thing to add.